File: test/cs/test_EvapoTranspiration.cs
'Test generation'

public static class Test
{
    //first);
    //test1

    public static void test1()
    {
        EvapoTranspiration res0 = Estimation_EvapoTranspiration.CalculateEvapoTranspiration(isWindVpDefined:1,evapoTranspirationPriestlyTaylor:449.367,evapoTranspirationPenman:830.957);

        Console.WriteLine("{0}","evapoTranspiration: "+res0.evapoTranspiration);

        Console.WriteLine("evapoTranspiration Comparison: ("+Math.Round(830.957,3)+";"+Math.Round(res0.evapoTranspiration,3)+") "+Equals(Math.Round(830.957,3),Math.Round(res0.evapoTranspiration,3)));
    }
}

File: test/java/test_EvapoTranspiration.java
//Test generation'

import java.math.BigDecimal;
public static class Test
{
    //first);
    //test1);

    public static void test1()
    {
        int isWindVpDefined = 1;
        double evapoTranspirationPriestlyTaylor = 449.367;
        double evapoTranspirationPenman = 830.957;

        EvapoTranspiration res0 = Estimation_EvapoTranspiration.CalculateEvapoTranspiration(isWindVpDefined,evapoTranspirationPriestlyTaylor,evapoTranspirationPenman);
        System.out.println(" evapoTranspiration: "+res0.evapoTranspiration);

        System.out.println(((new BigDecimal(res0.evapoTranspiration)).setScale(3, BigDecimal.ROUND_HALF_DOWN)).equals((new BigDecimal(830.957)).setScale(3, BigDecimal.ROUND_HALF_DOWN)));
    }
}
